frontend/src/app/components/NewsletterForm.js

```diff
@@ -0,0 +1,92 @@
+import classnames from 'classnames';
+import React from 'react';
+
+
+export default class NewsletterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.handlePrivacyClick = this.handlePrivacyClick.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
we aren't consistent in this repo wrt this vs `e => this.foo(e)` and should eventually decide on one or the other